test/rabbitmq_peer_discovery_etcd_SUITE.erl

```diff
@@ -94,42 +92,51 @@ extract_nodes_test(_Config) ->
     Expectation = ['rabbit@172.17.0.7', 'rabbit@172.17.0.5'],
   ?assertEqual(Expectation, rabbit_peer_discovery_etcd:extract_nodes(Values)).
 
-
 base_path_defaults_test(_Config) ->
     ?assertEqual([v2, keys, "rabbitmq", "default"],
                  rabbit_peer_discovery_etcd:base_path(#{})).
 
-
 base_path_custom_test(_Config) ->
-  C = #{etcd_prefix => <<"example/com/1.2.3/config/mq">>,
-        cluster_name => <<"test_cluster">>},
+    C = #{etcd_prefix => <<"example/com/1.2.3/config/mq">>,
+          cluster_name => <<"test_cluster">>},
     ?assertEqual([v2, keys, "example/com/1.2.3/config/mq", "test_cluster"],
                  rabbit_peer_discovery_etcd:base_path(C)).
 
+base_path_custom_test_with_slash(_Config) ->
+    C = #{etcd_prefix => <<"example/com/1.2.3/config/mq">>,
+          cluster_name => <<"test/cluster">>},
```

Comment:
That's an OK test but I somewhat doubt cluster names with slashes make sense. Key prefix paths with slashes make a lot of sense to me.